app/views/clients/rollup/_assessments.html.haml

```diff
@@ -77,7 +77,7 @@
           - any_buttons = assessments.any?
           - cz = "assessment_type_#{i}"
           %tr{ class: ( 'assessment__new-type' if any_buttons )}
-            %td= link_to name, client_hud_assessment_path(@client, assessment), class: 'btn btn-primary btn-muted btn-xs', data: { loads_in_pjax_modal: true }
+            %td= link_to_if can_view_enrollment_details_tab?, name, client_hud_assessment_path(@client, assessment), class: 'btn btn-primary btn-muted btn-xs', data: { loads_in_pjax_modal: true }
```

Comment:
I think there are other links that need this too. When I turned off the permission, it would open an empty modal and then redirect me somewhere.

As an aside, the permission description states `Users with Can View Clients get this automatically.` which doesnt seem to be true, since I wasn't able to view these details.

Reply:
Thanks!  pesky `&&` that should have been `||`